src/connectors/connector_pubsub.workflows.yaml

```diff
@@ -24,7 +24,8 @@
     - project: ${sys.get_env("GOOGLE_CLOUD_PROJECT_ID")}
     - topic: "[fill in a Pubsub topic name]"
     - subscription: "[fill in a Pubsub subscription name]"
-    - base64Data: "aGVsbG8="  # Some random base64 data.
+    - base64Data: ${base64.encode(json.encode(message))}  # Some random base64 data object.
```

Comment:
The `message` variable needs to be defined somewhere. We could create a `message` variable in the same assign list:
- message: {"hello": "world"}